unobot.py

```diff
@@ -360,15 +360,22 @@ def render_counts(self, full=NO):
 
     @staticmethod
     def render_cards(bot, cards, who):
+        cards = sorted(cards)
+        wilds = []
+        for card in cards:
```

Comment:
I'd like to go through in reverse, actually. Skip through the `Y` cards, then `while 'W' in card` and end the loop when there are no more `W` cards. Would be a nice way to avoid iterating through most of the deck, since it's sorted beforehand.

On the other hand, it might reduce iterations through the `cards` list more if the initial sort was just removed. Then strip out the `W` cards, sort the remaining cards and wilds separately, and concatenate them.

<dl>
<dt>With initial sort</dt>
<dd>2 full iterations (sort + for loop) _plus_ a variable fraction depending on how long the wilds list is. (This last sort might not be necessary, actually, since the list is already sorted in the first place.)</dd>
<dt>Without initial sort</dt>
<dd>2 iterations total: for loop + sorting the two separated parts of the list before recombining them.</dd>
</dl>